configuration/metadatatermmappings/metadataterms.csv

```diff
@@ -0,0 +1,7 @@
+Uuid,Void/Retire,Mapping code,Mapping source,Metadata class name,Metadata Uuid,_order:2000
```

Comment:
Defer to you as to whether all of these are correct and accurate.

Reply:
Yes, they are correct, I used them from our DB.